### PR TITLE
Do not grab devices less than 6 dof

### PR DESCRIPTION
**Pull request by tpiekar**
Had an issue with the driver grabbing my cad mouse when using my 3dconnexion wireless kit, which uses 1 receiver to connect spacemouse and cad mouse.

### CODE REVIEW

File: src/dev_usb_linux.c [hunk 1]
```diff
@@ -118,6 +118,12 @@ int open_dev_usb(struct device *dev)
 		}
 	}
 
+    // Lets ignore all devices with less than 6 axis to avoid the daemon to grab mice
```
Comment (jtsiomb) [inline src/dev_usb_linux.c]: Is this change really necessary? The user can explicitly select a usb id they want to use with spacenavd, and while I agree that it doesn't make sense to attempt to use any device which is not 6-dof, I'd rather not artificially restrict this. Is the other change in dev.c not sufficient to fix the issue?

Comment (tpiekar) [reply src/dev_usb_linux.c]: Hi. You were right. I removed the blacklisting and pushed the commit.

**Comment by jtsiomb** [comment]
Thank you, merged.
